rust/kernel/module_param.rs

```diff
@@ -0,0 +1,198 @@
+// SPDX-License-Identifier: GPL-2.0
+
+//! Types for module parameters.
+//!
+//! C header: [`include/linux/moduleparam.h`](../../../include/linux/moduleparam.h)
+
+use core::fmt::Write;
+
+/// Types that can be used for module parameters.
+/// Note that displaying the type in `sysfs` will fail if `to_string` returns
```

Comment:
Which `to_string()`? If you are referring to a C API, please mention it (ideally, add a link).